README.md

```diff
@@ -108,15 +108,6 @@ To run the project, execute the following command:
 
 --> Click on live Link or Click on live server in Vs Studio.
 
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-In order to run for linter errors, you will have to create a pull request
---->
-
 ### Deployment
 
```

Comment:
- [ ] Hi, I would advise you to Add in the Readme all that is needed for running the app. I am aware that this requirement was not mandatory. It is important to guide the user in how to Run your App it shows a lot of professionalism. 
- Here is what I would suggest: Under the section `Usage` in Readme, add  `npm install`, `npm run build`, and `npm run start`.